js-console/src/main/scala/org/typelevel/log4cats/console/ConsoleLoggerFactory.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats
+package console
+
+import cats.effect._
+import cats.syntax.all._
+
+class ConsoleLoggerFactory[F[_]: Sync] extends LoggerFactory[F] {
+  override def getLoggerFromName(name: String): SelfAwareStructuredLogger[F] = new ConsoleLogger[F]
+  override def fromName(name: String): F[SelfAwareStructuredLogger[F]] =
+    getLoggerFromName(name).pure[F].widen
+}
+
+object ConsoleLoggerFactory {
+  def apply[F[_]: Sync]: ConsoleLoggerFactory[F] = new ConsoleLoggerFactory[F]
```

Comment:
Since `LoggerFactory` is treated as a typeclass, then perhaps we should provide an implicit instance builder as we do for `Slf4jFactory` and make `ConsoleLoggerFactory` a trait.

Reply:
Addressed in 4cc405b4.